libraries/AP_Scripting/examples/ahrs-set-origin.lua

```diff
@@ -0,0 +1,16 @@
+-- example script for using "set_origin()"" and "initialized()"
+-- sets the ekf origin if not already set
+
+
+function update ()
+
+    if ahrs:initialised() then
+        origin = assert(not ahrs:get_origin(),"EKF origin already set")
+        location = Location() location:lat(-353632640) location:lng(1491652352) location:alt(58409)
+        ahrs:set_origin(location)
+    end
+
+    return update, 5000
```

Comment:
Whether successful or not, after setting the origin we don't need to call the script back right?
```suggestion
    return
```

Prevents getting the error from the assert when we were successful